front/src/modules/timesStops/helpers/formatScheduleData.ts

```diff
@@ -0,0 +1,23 @@
+import { ISO8601Duration2sec, datetime2sec, secToHoursString } from 'utils/timeManipulation';
+
+import type { ScheduleEntry } from '../types';
+
+function formatScheduleData(schedule: ScheduleEntry, startTime: string) {
+  if (!schedule) {
+    return { arrival: '', departure: '', stopFor: '' };
+  }
+  const startTimeSeconds = datetime2sec(new Date(startTime));
+  const arrivalSeconds = schedule.arrival ? ISO8601Duration2sec(schedule.arrival) : '';
+  const stopForSeconds = schedule.stop_for ? ISO8601Duration2sec(schedule.stop_for) : '';
+  const departure =
+    arrivalSeconds && stopForSeconds
+      ? secToHoursString(startTimeSeconds + arrivalSeconds + stopForSeconds, true)
+      : '';
+  return {
+    arrival: arrivalSeconds ? secToHoursString(startTimeSeconds + arrivalSeconds, true) : '',
+    departure,
+    stopFor: String(stopForSeconds), // I have some numbers in the input table state
```

Comment:
A comment written in first-person feels wrong

The value doesn't come from the input table state, right? It's defined at line 11 (parsed from a string)

Reply:
removed it as it’s no longer relevant